back/app/models/initiative.rb

```diff
@@ -110,6 +110,20 @@ class Initiative < ApplicationRecord
   scope :no_feedback_needed, -> { with_status_code(InitiativeStatus::CODES - ['threshold_reached']) }
   scope :proposed, -> { with_status_code('proposed') }
 
+  def cosponsor_ids=(ids)
+    return unless ids
+
+    ids = ids.uniq
+    current_ids = cosponsors.pluck(:id).uniq
+    return if current_ids.sort == ids.sort
+
+    cosponsors_initiatives.where.not(user_id: ids).destroy_all
+
+    (ids - current_ids).each do |id|
+      cosponsors_initiatives.create(user_id: id)
+    end
+  end
+
```

Comment:
@alexander-cit At the moment, if include the ID of a user that does not exist in the `ids` array, the request will fail with
```
"errors": {
        "cosponsors_initiatives": [
            {
                "error": "invalid"
            }
        ]
    }
```
Do we want this to happen? Or would we prefer a different (more useful) error? Or would we prefer that this method filters out such ids and then we simply do not attempt to create `cosponsors_initiatives` records with such invalid IDs?

Reply:
AFAIU it can happen only if the user exists when she's mentioned, and then deleted before the request is processed. Right?

In this case, it's very unlikely to happen, so I assume we can ignore it.